packages/zowe-explorer/src/uss/actions.ts

```diff
@@ -183,7 +183,7 @@ export async function uploadFile(node: IZoweUSSTreeNode, doc: vscode.TextDocumen
                 task,
             };
             if (prof.profile.encoding) {
-                options.localEncoding = prof.profile.encoding.toString();
+                options.encoding = prof.profile.encoding;
```

Comment:
Why is toString() no longer needed?

Reply:
zosmf encoding is passed as a number

Reply:
@kristinochka yes and the zosmf api putContent() is expecting a number for encoding where localEncoding was a string so the the number had to be converted to string but since Zowe CLI is looking for encoding vs localEncoding we will pass that.